.github/workflows/build-push.yml

```diff
@@ -177,7 +177,7 @@ jobs:
               "spec.template.spec.containers[0].image": "registry.nordix.org/eiffel/etos-iut:${{ needs.build_iut.outputs.iutVersion }}"
             },
             "manifests/base/executionspace/deployment.yaml": {
-              "spec.template.spec.containers[0].image": "registry.nordix.org/eiffel/etos-executionspace:${{ needs.build_executionspace.outputs.executionSpaceVersion }}"
+              "2.spec.template.spec.containers[0].image": "registry.nordix.org/eiffel/etos-executionspace:${{ needs.build_executionspace.outputs.executionSpaceVersion }}"
```

Comment:
I don't understand why we add "2." here

Reply:
The image reference is located in the second document of this YAML file: https://github.com/eiffel-community/etos-api/blob/main/manifests/base/executionspace/deployment.yaml  

I. e. after the first "---". This is different to all other deployment YAML files in the manifest/base directory.

Reply:
Please move the configmap part of the deployment.yaml file to its own file instead so that it matches the structure we have everywhere else.